### PR TITLE
Some cleanup in shmem reductions

### PR DESCRIPTION
**Pull request by boehms**

### CODE REVIEW

**Comment by manjugv** [comment]
@boehms Can you please resolve conflicts ?

**Comment by boehms** [comment]
done

File: content/shmem_reductions.tex [hunk 1]
```diff
@@ -200,24 +202,27 @@
...
     restored to the original values.
 
-    The sum and product reduction routines include complex-typed interfaces
-    for the \Cstd API only.
-    When the \Cstd translation environment does not support complex types%
-    \footnote{That is, under \Cstd language standards prior to \Cstd[99] or
-      under \Cstd[11] when \CONST{\_\_STDC\_NO\_COMPLEX\_\_} is defined to 1},
-    an \openshmem implementation is not required to provide support for
-    these complex-typed interfaces.
+    The sum and product reduction routines include complex-typed
```
Comment (manjugv) [inline content/shmem_reductions.tex:211]: @boehms Why did you remove this ?

Comment (boehms) [reply content/shmem_reductions.tex:211]: The fortran API is deprecated. We only have a C API going forward. To me it does not make sense to say that we do not support complex types in Fortran.
